--- parser/Makefile.orig	2014-02-21 03:48:37.352081391 +0100
+++ parser/Makefile	2014-02-21 04:08:18.004870123 +0100
@@ -0,0 +1,20 @@
+CC = gcc
+CFLAGS = -O2 -g
+
+.PHONY : all install clean
+
+all : zephir
+zephir : parser.c
+	$(CC) -w parser.c scanner.c -ljson-c -o ../bin/zephir-parser
+
+parser.c : lemon
+	re2c -o scanner.c scanner.re && ./lemon -s parser.lemon && cat base.c >> parser.c
+	sed s/"\#line"/"\/\/"/g scanner.c > xx && mv -f xx scanner.c
+	sed s/"#line"/"\/\/"/g parser.c > xx && mv -f xx parser.c
+lemon :
+	$(CC) -w lemon.c -o lemon
+	
+clean :
+	rm -f parser/*.o
+	rm -f parser/*.lo
+	
